### ISSUE TITLE
1803 Version Support

### ISSUE THREAD

**Issue by salroyal** [MicrosoftDocs/windowsserverdocs#925]
When will version 1803 be supported?

---
#### Document Details

⚠ *Do not edit this section. It is required for docs.microsoft.com ➟ GitHub issue linking.*

* ID: 29295bfb-4d36-07cc-4e47-29835aa835ec
* Version Independent ID: 618170e7-cc5c-2ff8-af3b-701c1b26ed11
* Content: [What type of installation is right for you](https://docs.microsoft.com/en-us/windows-server/manage/windows-admin-center/plan/installation-options)
* Content Source: [WindowsServerDocs/manage/windows-admin-center/plan/installation-options.md](https://github.com/MicrosoftDocs/windowsserverdocs/blob/master/WindowsServerDocs/manage/windows-admin-center/plan/installation-options.md)
* Product: **windows-server-threshold**
* GitHub Login: @nwashburn-ms
* Microsoft Alias: **niwashbu**

**Comment by jwwool** [comment]
Installation on Windows 10 1803 is supported, we will update the documentation.

**Comment by jwwool** [comment]
The documentation has been updated to clarify that Windows Admin Center can be installed on Windows 10 version 1709 or newer.

**Comment by christianacca** [comment]
Is windows *server* 1803 supported? Thanks

**Comment by jwwool** [comment]
@christianacca yes Windows Server version 1803 is supported- thanks for catching that it's not listed. We will update the documentation.

**Comment by tfletcher87** [comment]
Is Server 2016 1607 supported for install?

**Comment by jwwool** [comment]
As long as it's not Nanoserver, installation on Windows Server 2016 is supported.
